compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -3930,9 +3930,6 @@ private BLangIdentifier createIdentifier(DiagnosticPos pos, String value, Set<Wh
         }
 
         if (value.startsWith(IDENTIFIER_LITERAL_PREFIX)) {
-            if (!escapeQuotedIdentifier(value).matches("^[0-9a-zA-Z.]*$")) {
-                dlog.error(pos, DiagnosticCode.IDENTIFIER_LITERAL_ONLY_SUPPORTS_ALPHANUMERICS);
-            }
```

Comment:
Can remove `escapeQuotedIdentifier()` method as well

Reply:
removed in the last commit